### DISCUSSION TITLE
License of SCS Software

### DISCUSSION THREAD

**Discussion by guettli** [SovereignCloudStack/standards#76] [General]
Hi,

I saw that this repo uses the GPL license: https://github.com/SovereignCloudStack/generics

I guess you know that the GPL is viral.

I am missing an agreement of the SCS to a particular license.

I have concerns, that SCS won't be adapted by companies, if the code uses the GPL.

Related: https://www.cncf.io/blog/2017/02/01/cncf-recommends-aslv2/

**Comment by itrich** [comment]
Hi @guettli,

thank you for your interest in our project and your feedback on our license. We have covered various aspects of our license considerations here: https://scs.community/docs/contributor/license-considerations.html

I'll convert this issue into a discussion, if you still have a strong opinion on whether we should reconsider our initial considerations.

Best wishes,

Eduard

**Comment by guettli** [comment]
I think about getting active in the SCS area. But the above hyperlink changed my mind. 

The software in the CNCF environment is successful because they avoid GPL/AGPL. 

Nobody wants a virus. I know that Linux and git use the GPL, but this is different, since this usualy does not affect the code of the company.

I don't think SCS will be adopted if you use the AGPL.

**Reply by itrich** [reply]
I have migrated this issue to a discussion so that we have a better thread-view. I'm happy to continue this discussion and hopefully to find an approach on how you still may want to consider joining our community.

**Comment by batistein** [comment]
@garloff as I see we now use the Apache 2.0? So this could be closed right?

**Reply by itrich** [reply]
We still have some repositories with a copyleft license, e.g. [generics](https://github.com/SovereignCloudStack/generics) or the [openstack-health-monitor](https://github.com/SovereignCloudStack/openstack-health-monitor). I thus would advocate to keep this open and invite anyone to raise this again either in a team call or as Issue in the corresponding repository.
